nextflow_schema.json

```diff
@@ -585,11 +585,34 @@
                     "type": "string",
                     "default": "markduplicates",
                     "description": "Specify which tool to use for deduplication.",
-                    "help_text": "Sets the duplicate read removal tool. By default uses `markduplicates` from Picard. Alternatively an ancient DNA specific read deduplication tool `dedup` (Peltzer et al. 2016) is offered. The latter utilises both ends of paired-end data to remove duplicates (i.e. true exact duplicates, as markduplicates will over-zealously deduplicate anything with the same starting position even if the ends are different).\n\n> ⚠️ DeDup can only be used on collapsed (i.e. merged) reads from paired-end sequencing.",
+                    "help_text": "Sets the duplicate read removal tool. By default uses `markduplicates` from Picard. Alternatively an ancient DNA specific read deduplication tool `dedup` (Peltzer et al. 2016) is offered. The latter utilises both ends of paired-end data to remove duplicates (i.e. true exact duplicates, as markduplicates will over-zealously deduplicate anything with the same starting position even if the ends are different).\n\n> \u26a0\ufe0f DeDup can only be used on collapsed (i.e. merged) reads from paired-end sequencing.",
                     "enum": ["markduplicates", "dedup"]
                 }
             },
             "fa_icon": "fas fa-clone"
+        },
+        "mitochondrial_to_nuclear_ratio": {
+            "title": "Mitochondrial to Nuclear Ratio",
+            "type": "object",
+            "description": "Options for the calculation of ratio of reads to one chromosome/FASTA entry against all others.",
+            "default": "",
+            "help_text": "> If using TSV input, Mitochondrial to Nuclear Ratio calculation is calculated per deduplicated library (after lane merging)",
+            "fa_icon": "fas fa-balance-scale-left",
+            "properties": {
+                "run_mtnucratio": {
+                    "type": "boolean",
+                    "description": "Turn on mitochondrial to nuclear ratio calculation.",
+                    "help_text": "Turn on the module to estimate the ratio of mitochondrial to nuclear reads.",
+                    "fa_icon": "fas fa-balance-scale-left"
+                },
+                "mtnucratio_header": {
```

Comment:
If you update the parameter as I described above, make sure torename here.

Reply:
```suggestion
                "mitochondrion_name": {
```